Description: pin the AppArmor feature set to the Stretch's kernel one
 .
 Let's smooth UX on kernel upgrades and allow ourselves to update the AppArmor
 policy in a relaxed manner.
Bug-Debian: https://bugs.debian.org/879585
Forwarded: not-needed
Author: intrigeri <intrigeri@debian.org>

--- a/etc/apparmor/parser.conf
+++ b/etc/apparmor/parser.conf
@@ -60,3 +60,7 @@
 ## Adjust compression
 #Optimize=compress-small
 #Optimize=compress-fast
+
+## Pin feature set (avoid regressions when policy is lagging behind
+## the kernel)
+features-file=/usr/share/apparmor-features/features.Tails
